.github/workflows/ci-cassandra.yml

```diff
@@ -9,6 +9,18 @@ on:
 jobs:
   cassandra:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version:
+        - distribution: cassandra
+          major: 3.x
+          image: 3.11
+          schema: v001 v002 v003
+        - distribution: cassandra
+          major: 4.x
+          image: 4.0
+          schema: v004 v004 v004
```

Comment:
the repetition doesn't make sense

Reply:
Since we v001, v002, v003 for cassandra 3.11, I had to repeat it. We anyhow need 2 containers atleast since https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/integration/cassandra_test.go#L69-L71 is connecting to cassandra1 and https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/integration/cassandra_test.go#L88-L91 is connecting to cassandra2  so thought to keep all 3.